tests/support/cobs/issue.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type { RadiclePeer } from "@tests/support/peerManager.js";
+
+export async function createIssue(
+  peer: RadiclePeer,
+  projectFolder: string,
+  title: string,
+  description: string,
+  tags?: string[],
+): Promise<string> {
+  const issueTags = tags ? tags.map(tag => ["--tag", tag]).flat() : [];
+  const issueOptions: string[] = [
+    "issue",
+    "open",
+    "--title",
+    title,
+    "--description",
+    description,
+    ...issueTags,
+  ];
+  const { stdout } = await peer.rad(issueOptions, { cwd: projectFolder });
+  const match = stdout.match(/Issue {3}([a-zA-Z0-9]*)/);
+  if (!match) {
+    throw new Error("Not able to parse issue id");
+  }
+  return match[1];
+}
+
+export async function stateIssue(
```

Comment:
I think `changeIssueState` would be clearer.

Reply:
Done